members/_posts/2016-10-04-lewic.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: memberpage
+category: cmember
+logo : lewic.png
+
+name: 'Lewi Cleantech GmbH'
+adress1: 'Am Gaubach 37'
+adress2: 
+adress3: 
+adress4: 
+zipcode: '91369'
+city: 'Wiesenthau'
+region: ''
+country: 'Germany'
+phone: '+49 9191 62 55 250'
+email: info@lewi-cleantech.de
+web: https://lewi-cleantech.de
+twitter: 
+github: 
+
+---
+Wir unterstützen kleine und mittelständische Unternehmen ihr Energiesparpotenzial gewinn-steigernd auszuschöpfen.

```

Comment:
This let's me wonder: The whole webpage is in English, do you really want to have the member page in German? Feels a bit weird, but ultimately, it is your decision...

But as the first paragraph will be shown on the member list as well, I would say that at least this should be in English.